docs/usage/getting-started/running.md

```diff
@@ -208,6 +208,7 @@ Read the platform-specific docs to learn how to setup authentication on your pla
 - [Gitea and Forgejo](../modules/platform/gitea/index.md)
 - [github.com and GitHub Enterprise Server](../modules/platform/github/index.md)
 - [GitLab](../modules/platform/gitlab/index.md)
+- [SCM-Manager](../modules/platform/scmm/index.md)
```

Comment:
```suggestion
- [SCM-Manager](../modules/platform/scm-manager/index.md)
```

You renamed from `scmm` to `scm-manager`. Do we need to update the URL?

Please check my suggestion before applying. 😉